### ISSUE TITLE
mezzanine xss

### ISSUE THREAD

**Issue by Trex-tbag** [stephenmcd/mezzanine#1921]
Version: <=4.3.1
My English is not good, the report is translated by Google.
Recurring vulnerabilities:
Vulnerability url: `http://127.0.0.1:8000/admin/blog/blogpost/add/`
When adding a blog, use Burpsuite to capture the package, modify the title to `test<svg/onload=alert(1)>` and the content as `<svg>`
![21CB7D45-8A10-4E42-BCF6-F43BA73AB36B](https://user-images.githubusercontent.com/27653537/56572380-8a1a1680-65f1-11e9-8df2-1699f180bc22.png)
Return `http://127.0.0.1:8000/blog/` to trigger the xss
![9A9C3CCC-D286-4AB9-87C2-B3E9A3FF99B6](https://user-images.githubusercontent.com/27653537/56572416-97cf9c00-65f1-11e9-9e89-5243f0e3e2cf.png)
The cause of the vulnerability is due to the description_from_content function of core/models.py, line 184, where the value of title is called, resulting in xss
![8F2A6B92-DB86-42CA-9DC0-4D195CB64BBA](https://user-images.githubusercontent.com/27653537/56572426-9e5e1380-65f1-11e9-8dd9-7caa834d196c.png)

**Comment by kenbolton** [comment]
Is this what you are describing? https://nvd.nist.gov/vuln/detail/CVE-2018-16632

Resolved by https://github.com/stephenmcd/grappelli-safe/commit/cb1d459b2cb96be4ea8be33060da7874525510e0

**Comment by Trex-tbag** [comment]
> Is this what you are describing? https://nvd.nist.gov/vuln/detail/CVE-2018-16632
> 
> Resolved by [stephenmcd/grappelli-safe@cb1d459](https://github.com/stephenmcd/grappelli-safe/commit/cb1d459b2cb96be4ea8be33060da7874525510e0)

No, not the same, the trigger point of this xss is in the first line
https://github.com/stephenmcd/mezzanine/blob/master/mezzanine/blog/templates/blog/blog_post_list.html lin112
![image](https://user-images.githubusercontent.com/27653537/56624826-233a4300-666d-11e9-84ae-af03aa24e212.png)
